### ISSUE TITLE
Feature: Sensor added

### ISSUE THREAD

**Issue by carlsendk** [lukasroegner/homebridge-nello#19]
When someone ring, its trigger a sensor item in HomeKit, so you can do automation based in this.

**Comment by AlexanderBabel** [comment]
Hey there,

during the upcoming weekend I will implement your request. But it not a promise that it will be finished at the end of the weekend.

Greetings
Alex

**Comment by Martijndevriez** [comment]
this feature first plzz :). 

btw your homebridge plugin is awesome

**Comment by AlexanderBabel** [comment]
Hey there,

I updated the plugin and added the switch and a motion sensor. Feel free to try it:

```bash
npm i git+https://github.com/lukasroegner/homebridge-nello.git#develop
```

Greetings
Alex

**Comment by AlexanderBabel** [comment]
A documentation of the new config entries can be found here: https://github.com/lukasroegner/homebridge-nello/tree/develop

**Comment by AlexanderBabel** [comment]
Available with v0.4.0
